_harp/about/_company.md

```diff
@@ -9,3 +9,17 @@ We support ourselves entirely through voluntary contributions from our alumni, t
 We believe this to be a replicable model of training that is capable of addressing acute skills shortages, widening access to work opportunities, and offering life-changing personal empowerment free at the point of delivery and without ongoing subsidies.
 
 We are partnering with a number of organisations in the social enterprise and charity sectors to provide freelancing and employment opportunities for our graduates and to open new schools in the UK and abroad.
+
+## Our hiring rate
+
+Our employment success rate for students who have actively sought development work after graduation is 100%.
+
+Of the 52 students who studied with us in 2014/15 and who actively sought development work, all 52 are now working either freelance or in permanent roles. This is a reflection of both the quality of our graduates and the level of support we provide to them after they graduate.
+
+Meanwhile, the proportion of students who have actually decided to seek development work after they graduate has grown dramatically for us in the last two years. The transformation from only one student in our very first (FAC0) cohort to every student in the Autumn 2015 (FAC6) cohort is a reflection of the growing reputation of the programme and the serious intent of our students. 
```

Comment:
Maybe I'm just not getting this importance from the paragraph that explains it. I think it might be the 'seek development work' phrase - I reckon it's just not clear what that means.
Maybe 'decided to work as web developers after they graduate has grown dramatically in the last two years' ('seek' is something I use for only job-seekers. Might be a language barrier thing.